src/components/ChevronKnapp.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Chevron from 'nav-frontend-chevron';
+import styled from 'styled-components';
+
+const StyledChevronKnapp = styled.button`
+    background: transparent;
+    border: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0.25em;
+    cursor: pointer;
+    color: #0067c5
+`;
+
+const StyledChevronTekst = styled.span`
+    color: #0067c5;
+    box-sizing: border-box;
+    margin-bottom: 2px;
+    border: 2px solid transparent;
+    ${StyledChevronKnapp}:hover & {
+        border-bottom: 1px solid #0067c5;
+    }
+`;
+
+const StyledChevron = styled(Chevron)`
```

Comment:
Denne er tom

Reply:
Lurer på om dette måtte til for å referere til den inne i en annen styled component.